### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/code/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#956] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/code/demo.html

让你的 VuePress 站点中的 Markdown 文件支持代码案例。

https://theme-hope.vuejs.press/zh/guide/markdown/code/demo.html

**Comment by GodX-18** [comment]
js 中使用 document 会报错
